101-cosmosdb-api-for-mongodb-account-create/azuredeploy.json

```diff
@@ -0,0 +1,26 @@
+{
+	"$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+	"contentVersion": "1.0.0.0",
+	"parameters": {
+		"databaseAccountName": {
+			"type": "string",
+			"metadata": {
+				"description": "The MongoDB database account name."
```

Comment:
is this globally unique? if so, we should state as such...

Reply:
If the value the user needs to provide has to be globally unique across all of Azure, just add a note in the description to state that...  if it just needs to be unique in the resourceGroup then what you have is fine...

Reply:
Got it, just did the test, needs to be globally unique. Note about that added in the description of the associated parameter. Thanks!